islandora.module

```diff
@@ -167,6 +184,34 @@ function _islandora_fire_media_file_derivative_reaction(MediaInterface $media) {
   $utils->executeMediaReactions('\Drupal\islandora\Plugin\ContextReaction\DerivativeFileReaction', $media);
 }
 
+/**
+ * Helper function to reindex nodes after the referencing media changes.
+ */
+function _islandora_reindex_nodes(array $node_ids) {
+  if (!\Drupal::moduleHandler()->moduleExists('search_api')) {
+    return;
+  }
+  $nodes = \Drupal::entityTypeManager()
+    ->getStorage('node')
+    ->loadMultiple($node_ids);
+
+  // Re-save the node to trigger re-index.
+  foreach ($nodes as $node) {
+    try {
+      $node->save();
```

Comment:
Saving just to trigger an index does not seem appropriate, potentially inflating the size of the DB with many revisions being created for each media?

Reply:
Though it did not create visible revisions in the revisions tab. Nor did it change the updated date.